internal/libooniengine/ticker.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"context"
+	"time"
+)
+
+type tickerResponse struct {
+	ElapsedTime float64 `json:",omitempty"`
+}
+
+// runTicker emits a ticker event every second. It is a subtask
+// associated with all tasks.
+func runTicker(ctx context.Context, close chan any,
```

Comment:
Because we have a timeout associated with each WaitForNextEvent, we don't need a ticker.